cli/env_file.go

```
@@ -0,0 +1,24 @@
+package cli
+
+import (
+	"fmt"
+	"log/slog"
+	"path/filepath"
+
+	"github.com/joho/godotenv"
+)
+
+func loadEnvFromFile(path string) error {
+	if path != "" {
+		absolutePath, err := filepath.Abs(path)
+		if err != nil {
+			return fmt.Errorf("failed to get absolute path for %s: %v", path, err)
+		}
+		if godotenv.Load(path) != nil {
```

Let's not discard the error here and inject it in the log message below?